index.css

```diff
@@ -0,0 +1,88 @@
+body {
+    font-family: 'Courier New', Courier, monospace;
+    font-size: 1.6rem;
+    line-height: 1.5;
+    text-align: center;
+    color: #333;
+    margin: 0;  
+}
+
+h1 {
+    font-size: 4rem;
+    margin-bottom: 0;
+}
+
+h3 {
+    font-size: 3.24rem;
+}
+
+#headline {
+    margin:  50px 0;
+    text-align: center;
+}
+
+ul {
+    max-width: 500px;
+    margin: 0 auto 45px auto;
+    text-align: left;
+    line-height: 1.6;
+}
+
+li {
+    margin: 16px 0;
+    
+}
```

Comment:
Please remove unnecessary line space. Your code does not look professional if you leave unnecessary line space.